browser/profiles/brave_renderer_updater.cc

```diff
@@ -102,7 +109,11 @@ void BraveRendererUpdater::UpdateRenderer(
       default_wallet ==
       brave_wallet::mojom::DefaultWallet::BraveWalletPreferExtension;
 
+  PrefService* pref_service = profile_->GetPrefs();
+  bool de_amp_enabled = pref_service->GetBoolean(de_amp::kDeAmpPrefEnabled);
```

Comment:
can we make this the combined value of the pref plus the feature? And also create a utility method in the component that gives the result of both given the PrefService and also use that in DeAmpThrottle? Easy to make a mistake otherwise when checking.

Reply:
But then this would rely on components/de-amp. I thought we didn't want that, and hence the suggestion to just check using the PrefService.

Reply:
no, it would rely on components/de_amp/common :)

Reply:
sorry, actually this code wouldn't access it at all, the combined value would be sent to the renderer